keras_retinanet/bin/convert_model.py

```diff
@@ -43,6 +43,10 @@ def parse_args(args):
     parser.add_argument('--no-nms', help='Disables non maximum suppression.', dest='nms', action='store_false')
     parser.add_argument('--no-class-specific-filter', help='Disables class specific filtering.', dest='class_specific_filter', action='store_false')
     parser.add_argument('--config', help='Path to a configuration parameters .ini file.')
+    parser.add_argument('--nms-threshold', help='Value for non maximum suppression threshold', dest='nms_threshold', type=float, default=0.5)
```

Comment:
No need to define `dest`, these are already the defaults.

Reply:
Bit nitpicky, but could you add a period after the `help` sentence?

Reply:
Done